src/applications/survivors-benefits/config/chapters/06-financial-information/medicalExpenses/medicalExpensesPages.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import {
+  textUI,
+  textSchema,
+  radioUI,
+  radioSchema,
+  currencyUI,
+  currencySchema,
+  currentOrPastDateUI,
+  currentOrPastDateSchema,
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+
+const recipientTypeLabels = {
+  VETERAN: 'Veteran (only select if a last or burial expense)',
+  SURVIVING_SPOUSE: 'Surviving spouse',
+  VETERANS_CHILD: 'Veteran’s child',
+  OTHER: 'Other',
+};
+
+const frequencyLabels = {
+  MONTHLY: 'Once a month',
+  YEARLY: 'Once a year',
+  ONE_TIME: 'One-time',
+};
+
+function introDescription() {
+  return (
+    <div>
+      <p className="vads-u-margin-top--0">
+        We’ll now ask about medical or certain other expenses that aren’t
+        reimbursed.
+      </p>
+      <p>These types of expenses can include:</p>
+      <ul>
+        <li>
+          <strong>Recurring medical expenses</strong> that insurance doesn’t
```

Comment:
Can you check to make sure this should be a strong or just use the `vads-u-font-weight--bold` and a span?